models/DataObject/ClassDefinition/Data/Wysiwyg.php

```diff
@@ -131,7 +131,7 @@ public function getDataFromResource(mixed $data, DataObject\Concrete $object = n
      */
     public function getDataForQueryResource(mixed $data, DataObject\Concrete $object = null, array $params = []): ?string
     {
-        $data = $this->getDataForResource($data, $object, $params);
+        $data = $this->getDataForResource($data, $object, array_merge($params, ['no_sanitize' => true]));
```

Comment:
Should we maybe use `disable_sanitize` as the key here? 
Or just `'sanitize' => false` WDYT?

Reply:
using `'sanitize' => false` now.